frameworks/kafka/tests/client.py

```diff
@@ -93,7 +93,7 @@ def install(self, kerberos: sdk_auth.KerberosEnvironment=None) -> dict:
             "container": {
                 "type": "MESOS",
                 "docker": {
-                    "image": "elezar/kafka-client:4b9c060",
+                    "image": "samvantran/kafka-client:4.1.0",
```

Comment:
@elezar I made a [PR against your repo](https://github.com/elezar/dcos-clients/pull/1) to update the kafka-client. Feel free to build an updated image and I'd be perfectly okay w/ setting this back to `elezar/kafka-client:xx`

Reply:
Thanks @samvantran. I have merged your changes and rebuilt `elezar/kafka-client:deca3d0`. Note that the last part is the SHA of the repo after your merged changes, so let's rather switch to that for now. There is a JIRA ticket out to handle our Docker images more consistently.